apps/playground/src/app/components/forms/single-file-attachment/single-file-attachment.component.html

```diff
@@ -4,6 +4,7 @@
       formControlName="attachment"
       [maxFileSize]="maxFileSize"
       [validateFn]="validateFile"
+      [labelText]="'label text'"
```

Comment:
This doesn't need square brackets and additional inner quotes since the value is a string literal. Do this instead:

```
    <sky-file-attachment
      formControlName="attachment"
      labelText="label text"
      [maxFileSize]="maxFileSize"
      [validateFn]="validateFile"
      (fileChange)="reactiveFileUpdated($event)"
      (fileClick)="fileClick($event)"
```